inst/components/scss/sidebar.scss

```diff
@@ -54,14 +64,17 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--bslib-sidebar-icon-size)),
     border-right: var(--bslib-sidebar-vert-border);
     border-top-right-radius: 0;
     border-bottom-right-radius: 0;
-    background-color: var(--bslib-sidebar-bg);
     color: var(--bslib-sidebar-fg);
+    background-color: var(--bslib-sidebar-bg);
+    // TODO: background can be transparent to be dark mode adaptive, need opaque "subtle" variant
+    backdrop-filter: blur(5px);
```

Comment:
Is this still needed now that we have the `opacity: 0` thing?

```suggestion
```

Reply:
I decided to keep this, it looks better with the blur, even as the contents fade out.